.github/pull_request_template.md

```diff
@@ -0,0 +1,58 @@
+```
+INSTRUCTIONS:
+Go through each of these INSTRUCTIONS sections, address them, and then delete them
+```
+
+## Summary
+```
+INSTRUCTIONS:
+Add summary - one or two sentences explaining the purpose of this PR.
+```
+
+## Details
+```
+INSTRUCTIONS:
+Provide further details about the changes, or how they fit into the roadmap.
+You can delete this section if it's not useful.
+```
+
+## Testing
+```
+INSTRUCTIONS:
+Further details about the tests you've added or manually performed -
+you can delete this section if it's not useful.
+```
+
+## Update Instructions
```

Comment:
Suggst renaming to "Labeling" and add "Remove this section if above actions been made" at the end.